BUILDING.md

```diff
@@ -48,3 +48,16 @@ To build without executing tests:
 To find the Java binary in your environment, run the appropriate command for your operating system:
 * Linux/macOS: `which java`
 * Windows: `for %i in (java.exe) do @echo. %~$PATH:i`
+
+--------------------------
+A new test is done on Eclipse Neon.3 Release (4.6.3) which already includes the m2e plugins.
```

Comment:
This is not quite grammatically correct: "A new test is done on ..." — can you please clarify what you're trying to say?